tests/fastapi-utils/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.poetry]
+name = "test-fastapi-utils"
+version = "0.1.0"
+description = "Test of python fastapi-utils package"
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.6"
```

Comment:
[3.7 is the oldest supported version](https://www.python.org/downloads/); does it make sense to limit to that instead?

Reply:
Yes, that sounds good.

Reply:
In the mean time I don't think it matters much.